dynamic-plugins/imports/package.json

```diff
@@ -18,9 +18,9 @@
   "peerDependencies": {
     "@janus-idp/backstage-plugin-3scale-backend": "1.5.15",
     "@janus-idp/backstage-plugin-aap-backend": "1.6.15",
-    "@janus-idp/backstage-plugin-acr": "1.4.13",
+    "@janus-idp/backstage-plugin-acr": "1.7.11",
```

Comment:
this is the version in the 1.3 branch.

wouldn't it make sense to bump the versions in 1.2, release them, and THEN pick up the correct latest versions for that new 1.6.z release? see https://github.com/janus-idp/backstage-plugins/pull/2350/files#r1797190554

Reply:
I missed the version in other branches, your right that makes sense based on the current versioning across the branches. That is:

main branch 1.8.z (already moved to community-plugins repo)
release-1.3 branch 1.7.z (currently at 1.7.11, which is correct for this branch)
1.2.x branch Since 1.7.11 is used in 1.3, and 1.8.z is in the main branch, the 1.2.z branch should have a version that's lower than 1.7.z to avoid conflicts (Logically speaking, your right). Hence 1.6.4 in 1.2.z